packages/web/src/components/Header/Help.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from "classnames";
+import { DirectionalHint, IContextualMenuItem, PrimaryButton } from "@fluentui/react";
+import * as React from "react";
+import { useNavigate } from "react-router-dom";
+
+import styles from "./Help.module.css";
+
+interface Props {
+    path: string;
+}
+
+// Banner for the splash page
+export default function HelpMenu(props: Props) {
+    const isApp: boolean = props.path == "/app";
```

Comment:
(nit) `isAppPage` or `isAppRoute` would be my slight preference to be slightly more specific